05/html/index.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<link rel="stylesheet" href="../css/global.css" />
+		<link rel="stylesheet" media="(max-width:768px)" href="../css/mobile.css" />
+		<link
+			rel="stylesheet"
+			media="(min-width:768px)"
+			href="../css/desktop.css"
+		/>
+		<title>Document</title>
+	</head>
+	<body>
+		<header>
+			<nav class="nav__menu">
```

Comment:
Jeśli mamy Element z BEM (`.nav__menu`) to w strukturze powinien pojawić się Block z BEM (`.nav`) - tutaj go brakuje.
Więcej znajdziesz w tym artykule:  https://devmentor.pl/b/metodologia-bem-w-css-i-sass